src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,13 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
+import java.io.File
+
 import scala.util.Try
 
 object EasyFileMetadata {
   def apply(s: FileItemSettings): Try[String] = Try {
+      val parentPath = s.pathInDataset.get.getParentFile
```

Comment:
Okay, I think I am getting it now. I am still rather vague about how we would get inside two `Option`s (`s.pathInDataset` and `s.title`) at once. Remind me next to have a chat about this. That'll be more convenient that doing it here in the pull request comments.